pkg/blob/controllerserver_test.go

```diff
@@ -164,18 +165,6 @@ func TestCreateVolume(t *testing.T) {
 		name     string
 		testFunc func(t *testing.T)
 	}{
-		{
```

Comment:
why this ut is removed?

Reply:
The deleted uts are cherry-picks from https://github.com/kubernetes-sigs/blob-csi-driver/commit/ad9d27c27e3770e857a81a10d0e887b9f246a899#diff-0b174fc042381a284b07f142da447934352f84b42ab6fb5c3cc5082857b376b5